DigitalOffice/Shared/Kernel/Exceptions/BaseException.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace Kernel.Exceptions
+{
+    /// <summary>
+    /// Indicates client or server error.
+    /// </summary>
+    public abstract class BaseException : Exception
+    {
+        /// <summary>
+        /// Status code are issued by a server in response to a client's request made to the server.
+        /// </summary>
+        public virtual int StatusCode { get; }
+        
+        /// <summary>
+        /// An explanatory phrase in English that explains to a person the reason for just such an answer.
```

Comment:
**Explanatory** phrase **explains**? Don't understand what does "just such an answer" mean.

Reply:
Yes. Explains why exactly this error came.

Reply:
Not sure if you need to specify what in English.

Reply:
Why not?